docs: improve README structure, clarity, and usability (#13569)
kyverno/kyverno: README.md

[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fkyverno%2Fkyverno.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fkyverno%2Fkyverno?ref=badge_shield)

<a href="https://kyverno.io" rel="kyverno.io"><img src="img/Kyverno_Horizontal.png" alt="Kyverno Logo" width="400"></a>

## 📑 Table of Contents

- [About Kyverno](#about-kyverno)
- [Documentation](#-documentation)
- [Demos & Tutorials](#-demos--tutorials)
- [Popular Use Cases](#-popular-use-cases)
- [Getting Help](#-getting-help)
- [Contributing](#-contributing)
- [Software Bill of Materials](#software-bill-of-materials)
- [Community Highlights](#-community-highlights)
- [Contributors](#contributors)
- [License](#license)

## About Kyverno

Kyverno is a Kubernetes-native policy engine designed for platform engineering teams. It enables security, compliance, automation, and governance through policy-as-code. Kyverno can:

- Validate, mutate, generate, and clean up resources using Kubernetes admission controls and background scans.
- Verify container image signatures for supply chain security.

> - Operate directly with Kubernetes-native tools like `kubectl`, `kustomize`, and Git—no new language required.

Reviewer: Proposed a suggestion because it feels like the previous emphasis was on the "no-new-tool" part of the sentence more than the "cloud/kubernetes-native" part.

```suggestion
- Operate with tools you already use — like `kubectl`, `kustomize`, and Git
```
